Quicksort_3_way/Quicksort_3_way.c

```diff
@@ -0,0 +1,104 @@
+    #include <stdio.h>
+    #include <stdlib.h>
+    #include <time.h>
+
+    typedef void qsort3way_swap(void *a, void *b);
+    typedef int qsort3way_cmp(void const *a, void const *b);
+
+     static void qsort3way_aux(char *array_begin, char *array_end, size_t size,
```

Comment:
please remove the spaces at the beginning

Reply:
@asha15  done 
look if anything more has to be done